python/ray/tests/test_state_api.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+import ray
+from ray.experimental.state.api import (
+    list_actors,
+    list_placement_groups,
+    list_nodes,
+    list_jobs,
+    list_workers,
+)
+from ray._private.test_utils import wait_for_condition
+from ray.job_submission import JobSubmissionClient
+
+
+def test_list_actors(shutdown_only):
+    ray.init()
+
+    @ray.remote
+    class A:
+        pass
+
+    a = A.remote()  # noqa
+
+    def f():
+        state = list(list_actors().values())[0]["state"]
```

Comment:
Could we also check to make sure actor id is correct?

Reply:
yep! will add tests. I haven't added that yet